ddtrace/contrib/grpc/aio_client_interceptor.py

```diff
@@ -160,13 +191,9 @@ async def _wrap_stream_response(
         span: Span,
     ) -> ResponseIterableType:
         try:
+            _handle_add_callback(call, _done_callback(span))
```

Comment:
As we discussed, we could also check if we are done with the iteration, similar to what we have here:

https://github.com/DataDog/dd-trace-py/blob/5148d18ff13f25914ea3bf5efbae9f6695ee3d64/ddtrace/contrib/grpc/client_interceptor.py#L146-L150

If we do that, I wonder if we could check get the code and details when we catch the `StopIteration`. What do you think?

Reply:
AFAIK only direct reads will result in a `StopIteration` so we cannot always rely on it for code and details. But i can add that behavior at least to catch the direct read case.